app/models/location.rb

```diff
@@ -15,6 +15,22 @@ class Location < ApplicationRecord
   LOCATION_TYPE_HIGH_SECURITY_HOSPITAL = 'high_security_hospital'
   LOCATION_TYPE_IMMIGRATION_DETENTION_CENTRE = 'immigration_detention_centre'
 
+  enum location_type: {
+    court: LOCATION_TYPE_COURT,
+    police: LOCATION_TYPE_POLICE,
+    prison: LOCATION_TYPE_PRISON,
+    secure_training_center: LOCATION_TYPE_SECURE_TRAINING_CENTER,
+    secure_childrens_home: LOCATION_TYPE_SECURE_CHILDRENS_HOME,
+    youth_offending_institute: LOCATION_TYPE_YOUTH_OFFENDNG_INSTITUTE,
+    approved_premises: LOCATION_TYPE_APPROVED_PREMISES,
+    probation_office: LOCATION_TYPE_PROBATION_OFFICE,
+    community_rehabilitation_company: LOCATION_TYPE_COMMUNITY_REHABILITATION_COMPANY,
+    foreign_national_prison: LOCATION_TYPE_FOREIGN_NATIONAL_PRISON,
+    voluntary_hostel: LOCATION_TYPE_VOLUNTARY_HOSTEL,
+    high_security_hospital: LOCATION_TYPE_HIGH_SECURITY_HOSPITAL,
+    immigration_detention_centre: LOCATION_TYPE_IMMIGRATION_DETENTION_CENTRE,
```

Comment:
Adding the `enum` is great  👍 
also, what about making the constants less verbose? considering they are already inside the `Location` class, would be nice to remove the `LOCATION_` path, and put TYPE at the end, for example:
`COURT_TYPE`,
`IMMIGRATION_DETENTION_CENTRE_TYPE`

Reply:
Hmm.. would rather not change all of the constants, they're used in too many places and would make the PR noisy, plus this is a convention used in other models.